sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2018,6 +2019,8 @@ def __init__(
         determined number of shards to write to BigQuery. This can be used for
         all of FILE_LOADS, STREAMING_INSERTS, and STORAGE_WRITE_API. Only
         applicable to unbounded input.
+      num_storage_api_streams: If set, the Storage API sink will default to
+        using this number of write streams. Only applicable to unbounded data.
```

Comment:
num_storage_api_streams: specifies the number of write streams that the Storage API sink will use. This parameter is only applicable to unbounded data.

Reply:
Shall we check this should be always set now for the unbounded data since it won't work otherwise.

Reply:
>Shall we check this should be always set now for the unbounded data since it won't work otherwise.

streaming writes with at-least-once still works without setting this parameter

Reply:
changed the documentation, thanks for the suggestion!

Reply:
I see. Shall we add something like "This parameter must be set for Storage API writes with the exactly once method."?

Reply:
I hesitate on doing this because conventionally we don't do runner-based checks in the SDK.

Reply:
I clarified it in the public BigQuery connector doc (https://beam.apache.org/documentation/io/built-in/google-bigquery/)